core/arch/arm/mm/core_mmu.c

```diff
@@ -431,14 +431,51 @@ unsigned int cache_maintenance_l1(int op, void *va, size_t len)
 	return TEE_SUCCESS;
 }
 
-__weak unsigned int cache_maintenance_l2(int op __unused,
-			paddr_t pa __unused, size_t len __unused)
+#ifdef CFG_PL310
+#include <kernel/tz_ssvce_pl310.h>
```

Comment:
In another patch you agreed with me to have the include at the top of the file. I'd guess that would make sense here also? (However in this case I think we should still put the under the ifdef).